src/get-workspaces-paths.js

```diff
@@ -1,8 +1,6 @@
 'use strict';
 
 const execa = require('execa');
-const { promisify } = require('util');
-const glob = promisify(require('glob'));
```

Comment:
Are we using this elsewhere? Should we remove it from package.json?

Reply:
nope, I removed it